### PR TITLE
Use worldedit to regenerate chunks.

### PR DESCRIPTION
**Pull request by dordsor21**
Use worldedit to regenerate chunks. It's very slow, but it works.

this should be used as a quick temp fix

### CODE REVIEW

File: Bukkit/src/main/java/com/github/intellectualsites/plotsquared/bukkit/util/block/BukkitLocalQueue.java [hunk 1]
```diff
@@ -64,7 +70,22 @@ public BukkitLocalQueue(String world) {
     @Override public final void regenChunk(int x, int z) {
         World worldObj = Bukkit.getWorld(getWorld());
         if (worldObj != null) {
-            worldObj.regenerateChunk(x, z);
+            try {
+                worldObj.regenerateChunk(x, z);
+            } catch (UnsupportedOperationException e) {
+                com.sk89q.worldedit.world.World world = BukkitAdapter.adapt(worldObj);
+                EditSession editSession =
```
Comment (wizjany) [inline Bukkit/src/main/java/com/github/intellectualsites/plotsquared/bukkit/util/block/BukkitLocalQueue.java]: you can just try-with-resources the whole editSession and let it close itself
```java
try (EditSession editSession = ...) {
   world.regen...
}
```

**Comment by Sauilitired** [comment]
How *slow* is this? As in, is a good idea to have it enabled by default, or should it be opt-in?

**Comment by dordsor21** [comment]
> How _slow_ is this? As in, is a good idea to have it enabled by default, or should it be opt-in?

Currently it's this or it doesn't work

**Comment by Sauilitired** [comment]
> > How _slow_ is this? As in, is a good idea to have it enabled by default, or should it be opt-in?
> 
> Currently it's this or it doesn't work

I'm aware, but I think some server owners would choose not having plot clearing at all over having it freeze their entire server.

**Comment by Sauilitired** [comment]
After more consideration, it's probably better for server owners to instead just limit clearing using permissions. It being slow is better than it being broken.

**Pull request merged**